src/symfony-bundle/src/Aeon/Symfony/AeonBundle/Validator/Constraints/After.php

```diff
@@ -10,9 +10,29 @@ final class After extends AbstractComparison
 {
     public const BEFORE_OR_EQUAL_ERROR = '99f63b74-a275-4a01-8678-63124971bff8';
 
+    /**
+     * @var array<string, string>
+     */
+    protected const ERROR_NAMES = [
+        self::BEFORE_OR_EQUAL_ERROR => 'BEFORE_OR_EQUAL_ERROR',
+    ];
+
+    /**
+     * @var array<string, string>
+     */
     protected static $errorNames = [
         self::BEFORE_OR_EQUAL_ERROR => 'BEFORE_OR_EQUAL_ERROR',
     ];
 
-    public $message = 'This value should be after {{ compared_value }}.';
+    public function __construct(mixed $value = null, string $propertyPath = null, string $message = 'This value should be after {{ compared_value }}.', array $groups = null, mixed $payload = null, array $options = [])
```

Comment:
I decided to use a constructor due to compiler issues when declaring that property on the class as it's also declared on the parent class leading to issues between Symfony versions.

Reply:
What is the problem with keeping this as a property?

Reply:
In version 5.4/6.4 the property is a `public $message`, but in 7.0 it's a `public string $message` thus, we cannot use either of it to support both, as if we declare it with type, it will lead to fatal in the older version, without it will do same but in newer.